Symbology/web/VBET/vbet_active_floodplain.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://xml.riverscapes.xyz/Symbology/web/vector.schema.json",
+  "legend": [
+    ["hsl(118, 58%, 69%)", "Estimated Active Floodplain"]
+  ],
+  "layerStyles": [
+    {
+        "id": "vbet-active-floodplain-b076i6",
+        "type": "fill",
+        "source": "composite",
+        "source-layer": "vbet_active_floodplain-b076i6",
+        "layout": {},
+        "paint": {"fill-color": "hsl(118, 58%, 69%)", "fill-opacity": 0.7}
+    }
+  ]
+}
```

Comment:
@MattReimer Is this the same thing happening here? Do I just need to go through and get rid of this (and it looks like 4 others) curly bracket?

Reply:
Vscode should find these problems for you so check the problems tab of the terminal to see if they check out. They should also show up as little red squiggles like a spellcheck if there's a problem

Reply:
Okay, not seeing issues beyond that one in `vbet_active_channel` json. Hopefully I'm understanding correctly.

Reply:
That just means that particular commit broke. Everything looks good to me.